spec/system/passenger_management_spec.rb

```diff
@@ -10,48 +10,67 @@
       @verifying_agency = create :verifying_agency
       when_current_user_is(@user)
     end
-    context 'creating a new passenger successfully' do
-      it 'creates the passenger' do
-        date = 2.days.since.strftime '%Y-%m-%d'
+    context 'creating a new passeger' do
+      it 'allows you to select a carrier if subscribed to sms' do
         visit passengers_path
         click_on 'Add New Passenger'
-        fill_in 'Passenger Name', with: 'Foo Bar'
-        fill_in 'Email', with: 'foobar@invalid.com'
-        fill_in 'Address', with: '123 turkey lane'
-        fill_in 'Phone', with: '123'
-        fill_in 'Passenger Spire', with: '12345678@umass.edu'
-        fill_in 'How long will the passenger be with us?', with: date
-        select @verifying_agency.name, from: 'Which agency verifies that this passenger needs rides?'
-        click_button 'Submit'
-        expect(page).to have_text 'Passenger successfully created.'
+        expect(page).not_to have_text 'Carrier'
+        check 'Subscribed to sms'
+        expect(page).to have_text 'Carrier'
       end
-      it 'checks for existing passengers if a duplicate spire is found' do
-        visit passengers_path
-        click_on 'Add New Passenger'
-        fill_in 'Passenger Spire', with: "#{@passenger.spire}\t"
-        expect(page).to have_text 'A passenger already exists for this Spire ID'
-        expect(page).to have_button 'Add new passenger'
-        expect(page).to have_link 'Edit existing passenger'
-      end
-    end
-    context 'creating a new passenger unsuccessfully' do
-      it 'renders spire errors in the flash' do
-        visit passengers_path
-        click_on 'Add New Passenger'
-        fill_in 'Passenger Spire', with: 'invalid spire'
-        click_button 'Submit'
-        expect(page).to have_text 'Spire must be 8 digits followed by @umass.edu'
+      context 'passenger creation successful' do
+        it 'creates the passenger' do
+          create :carrier
+          date = 2.days.since.strftime '%Y-%m-%d'
+          visit passengers_path
+          click_on 'Add New Passenger'
+          fill_in 'Passenger Name', with: 'Foo Bar'
+          fill_in 'Email', with: 'foobar@invalid.com'
+          fill_in 'Address', with: '123 turkey lane'
+          fill_in 'Phone', with: '123'
+          # Carrier selectbox has no blanks
+          check 'Subscribed to sms'
```

Comment:
Why do we need to check subscribed here? What do you mean by "carrier selectbox has no blanks"?
You should leave a test that creates a passenger with the absolute minimum necessary to create a passenger. That is, create one without being subscribed to text messages. 
Your first context should be "creating a new passenger that is subscribed to SMS", leaving the other success test alone.